docs/glossary.md

```diff
@@ -0,0 +1,26 @@
+| performance mode         | In performance mode, the app measures only the throughput.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| accuracy mode            | In accuracy mode, the app measures the throughput and the accuracy.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| LoadGen (Load Generator) | The LoadGen is a traffic generator for MLPerf Inference that loads the SUT and measures performance. Its behavior is controlled by a configuration file it reads at the start of the run. The LoadGen produces the query traffic according to the rules of the previously described scenarios (i.e., single-stream, multistream, server, and offline). Additionally, it collects information for logging, debugging, and postprocessing the data. It records queries and responses from the SUT, and at the end of the run, it reports statistics, summarizes the results, and determines whether the run was valid.<br>Source: <https://arxiv.org/pdf/1911.02549.pdf> |
+| single stream scenario   | The single-stream scenario represents one inference-query stream with a query sample size of 1, reflecting the many client applications where responsiveness is critical. An example is offline voice transcription on Google’s Pixel 4 smartphone. To measure performance, we inject a single query into the inference system; when the query is complete, we record the completion time and inject the next query. The metric is the query stream’s 90th-percentile latency.<br>Source: <https://arxiv.org/pdf/1911.02549.pdf>                                                                                                                                       |
```

Comment:
>The metric is the query stream’s 90th-percentile latency.

I think we should note that in our application the latency is converted to equivalent throughput.